README.md

```diff
@@ -27,12 +39,12 @@ a benchmark using the [Swarm64 TPC Toolkit](https://github.com/swarm64/tpc-toolk
    The provided default configuration is suited to run benchmarks up to 1TB worth 
    of data and 384GB of system RAM.
 
-3. To start a database instance, call `docker-compose` accordingly:
+4. To start a database instance, call `docker-compose` accordingly:
 
    - S64 DA on Intel PAC Arria10: `docker-compose -f docker-compose-s64da-pac.yml up`
    - S64 DA on Xilinx Alveo U200: `docker-compose -f docker-compose-s64da-xilinx.yml up`
    - Native PostgreSQL: `docker-compose -f docker-compose-psql.yml up`
 
-4. Connect to the instance: `psql -h localhost -U postgres`
+5. Connect to the instance: `psql -h localhost -U postgres`
```

Comment:
How about "-p 5432" here as well to show what is required when more that one DB instance is running as mentioned.